### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by gghandsfield**
created integration application to bring together input and output information

### CODE REVIEW

File: source/drhtaxes/application.py [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-
-import sys
-
+from drhtaxes.individual import Tax
 from drhtaxes.admission import admission
+from drhtaxes.corporate import COR
+from drhtaxes.output import tax_compare
 
 
 def main():
```
Comment (nickerso) [inline source/drhtaxes/application.py:8]: A bit of a comment describing this function would be useful.

**Review by nickerso — CHANGES_REQUESTED** [review]
Looks great, just missing a bit of documentation.

File: source/drhtaxes/application.py (hunk 1 above)
Comment (gghandsfield) [reply source/drhtaxes/application.py:8]: Sorry nickerso, I added this as a new commit

**Review by nickerso — APPROVED** [review]
Looks great to me!

**Pull request merged**